movethebytes/data-transfer/rust/_compositions/quic.toml

```diff
@@ -0,0 +1,22 @@
+[metadata]
+  name = "quic"
+
+[global]
+  plan = "rust"
+  case = "quic"
+  builder = "docker:generic"
+  runner = "local:docker"
+  total_instances = 2
+  disable_metrics = false
+
+[[groups]]
+  id = "groups1"
+  [groups.build_config.build_args]
+  BINARY_NAME = "quic"
+  [groups.instances]
+    count = 2
+  [groups.run]
+    [groups.run.test_params]
+      latency = "100ms"
+      bandwidth = "10MiB"
+      bslatency="100ms"
```

Comment:
let's drop bslatency, don't think we currently use it + it just simulates processing latency which is useful for "faked out" versions of bitswap or whatever impl, not really useful for real implementations.